src/com/example/ui.cljs

```diff
@@ -22,36 +22,38 @@
    :ident (fn [] [:component/id :header])
    :initial-state {}}
   (let [load-user (get props [df/marker-table :load-progress])]
-    (div :.flex.justify-between.items-center
-         (div :.text-zinc-800 "Should I Train?")
+    (div :.flex.justify-between.items-center.my-4
+         (div :.text-grey-100 "Should I Train?")
          (when-not load-user
            (div :.flex.flex-end
                 (when (:user/authenticated? user) (div :.flex
-                                                       (p :.mr-2 (str "Hello " (or (:user/name user) "from the ui/Root component") "!"))
-                                                       (button {:onClick #(on-signout)} "Sign out")))
-                (when (not (:user/authenticated? user)) (button {:onClick #(github-signin)} "Sign in with GitHub")))))))
+                                                       (p :.mr-2.text-grey-100 (str "Hello " (or (:user/name user) "from the ui/Root component") "!"))
+                                                       (button :.text-grey-100 {:onClick #(on-signout)} "Sign out")))
+                (when (not (:user/authenticated? user)) (button :.text-grey-100 {:onClick #(github-signin)} "Sign in with GitHub")))))))
 
 (def ui-header (comp/factory Header))
 
 (defn field [{:keys [label valid? error-message input-class id] :as props}]
   (let [input-props (-> props
                         (dissoc :label :valid? :error-message :input-class :options))]
     (div
-     (dom/label {:htmlFor label} label)
+     (dom/label :.relative.block.bg-white.border.rounded-lg.shadow-sm.px-6.py-4.cursor-pointer.sm:flex.sm:justify-between.focus:outline-none.text-gray-700 {:htmlFor label} label)
      (input-class input-props)
```

Comment:
This `input-class` is giving me trouble. Not sure how to style the radio button. Ideally it would move in the label or be hidden entirely

Reply:
you have to add `appearance: none` and then add all your custom properties. This looks like a good guide https://moderncss.dev/pure-css-custom-styled-radio-buttons/